.github/workflows/stale-issues.yml

```
@@ -0,0 +1,23 @@
+name: Send comment on stale issues
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
```

Another super nitpick:

```suggestion
  stale:
    name: Stale
```